docs/getting_started/tutorial.md

```diff
@@ -248,17 +213,18 @@ $ python3 manage.py runserver
 
 Search for **Event Subscriptions** on your app's **Basic Information** page and toggle the `Enable events` button.
 You'll be asked to type a **Request URL**. Get the generated URL by ngrok (the one that that starts with https://)
-and append `/slack/events` to it. 
-For example, if your generated URL is something like https://e54acd3a20b3.ngrok.io. Then, the **Request URL** you
-should enter in Slack should be https://e54acd3a20b3.ngrok.io/slack/events.
+and append `/wagtail_live/slack/events` to it. 
+For example, if your generated URL is something like https://e54acd3a20b3.ngrok.io, then the **Request URL** you
+should enter in Slack should be 
+https://e54acd3a20b3.ngrok.io/wagtail_live/slack/events.
 
 As soon as you type the URL, Slack will send a POST request to verify the URL given with a challenge parameter.
-You don't have to bother about it, Bolt handles it.
+You don't have to bother about it, Wagtail Live handles it.
 
 ### Channel configuration
 
 After your request URL is verified, scroll down to **Subscribe to Bot Events** and click the `Add Bot User Event` button.
-Choose the `message.channels` event and hit the `Save Changes` button. This allow your bot to listen for messages in public channels that it is added to.
+Choose the `message.channels` event and hit the `Save Changes` button. This allow your bot to listen for messages in **public channels** that it is added to.
```

Comment:
**Question:** it is not possible to have publish content from private Slack channels?

Reply:
Yes, it's possible. You have to add the right scope, see this https://github.com/wagtail/wagtail-live/blob/tutorial/docs/getting_started/setup_slack.md#subscribe-to-bot-events